src/Isomorphisms.agda

```diff
@@ -1,570 +1,49 @@
-module Isomorphisms where
+{-# OPTIONS --allow-unsolved-metas #-}
+
+open import Categories
+
+module Isomorphisms (C : Category) where
```

Comment:
Notice that the whole module is indexed by a category, which is the category we are defining isomorphisms in.